### PR TITLE
Fixed decorator in LibtorrentDownloadImpl

### PR DESCRIPTION
**Pull request by devos50**
@ardhipoetra when writing additional tests, I noticed that a checkHandleAndSynchronize decorator missed parenthesis.

### CODE REVIEW

**Comment by whirm** [comment]
@ardhipoetra double check that you didn't leave any more of those behind please

**Comment by devos50** [comment]
retest this please

**Comment by devos50** [comment]
@lfdversluis can you review?

**Comment by ardhipoetra** [comment]
Yes actually I noticed this and fixed this in #2399

**Comment by devos50** [comment]
@ardhipoetra great! In the future, please create a separate PR for this so it can get merged earlier :)

**Comment by lfdversluis** [comment]
Looks ok

**Comment by devos50** [comment]
Great, @whirm can you review?

File: Tribler/Test/Core/Libtorrent/test_libtorrent_download_impl.py [hunk 1]
```diff
@@ -7,12 +7,13 @@
...
 from Tribler.Core.Utilities.configparser import CallbackConfigParser
 from Tribler.Core.Utilities.twisted_thread import deferred, reactor
+from Tribler.Test.Core.base_test import TriblerCoreTest, MockObject
 from Tribler.Test.test_as_server import TestAsServer, TESTS_DATA_DIR
 
 
 class TestLibtorrentDownloadImpl(TestAsServer):
     """
-    This class provides unit tests tha tes tthe LibtorrentDownloadImpl class.
+    This class provides unit tests tha test the LibtorrentDownloadImpl class.
```
Comment (whirm) [inline Tribler/Test/Core/Libtorrent/test_libtorrent_download_impl.py]: you missed one of the 3 typos in a row (tha->that)

**Comment by whirm** [comment]
Otherwise looks fine, fix/squash and I'll merge

**Comment by devos50** [comment]
@whirm done

**Comment by whirm** [comment]
Those changes are on the PR I just reviewed too

**Comment by devos50** [comment]
@whirm true, I cherry-picked commits in this PR in the other PR. Can you merge this one first? Then I will remove the commits from the other PR.

**Comment by devos50** [comment]
@whirm I didn't notice this PR was to `next`. I need the changes in this PR for #2422. I can either wait for #2439 to be merged or I will cherry-pick these commits again in #2422. What do you want?

**Comment by whirm** [comment]
I'll do it, don't worry

**Pull request merged**